src/language/server.ts

```diff
@@ -122,7 +121,7 @@ const getDocumentSettings = (resource: string): Thenable<ExampleSettings> => {
   if (!result) {
     result = connection.workspace.getConfiguration({
       scopeUri: resource,
-      section: 'mongodbLanguageServer'
+      section: 'mongodbLanguageServer',
```

Comment:
You added trailing commas everywhere to this file :)

Reply:
ahhhhhhhhhh hah. thanks